### PR TITLE
update zouwu README with autots and built-in model comparison

### PR DESCRIPTION
**Pull request by shane-huang**
* some fixes in input shape descripitons
* rewording 
* add comparison between built-in models and AutoTS

### CODE REVIEW

**Comment by yushan111** [comment]
may also need:

- psutil

- scikit-learn>=0.20.0

- requests 

- bayesian-optimization

in requirements.

**Comment by shane-huang** [comment]
> may also need:
> 
> * psutil
> * scikit-learn>=0.20.0
> * requests
> * bayesian-optimization
> 
> in requirements.

Do we really need bayesian-optimization now? I think ray tune uses bayesopt

File: pyzoo/zoo/zouwu/README.md [hunk 1]
```diff
@@ -156,3 +163,24 @@ AutoTSTrainer and TSPipeline accepts data frames as input. An exmaple data frame
  # ... do sth. e.g. incremental fitting
  loaded_ppl.save(another_file)
  ```
+### built-in models vs. AutoTS
+
+Now we show some comparison results between manually tuned models vs. AutoTS, using network traffic forecasting as an example.
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/README.md:168]: add a link to network traffic forecasting use case

File: pyzoo/zoo/zouwu/README.md [hunk 2]
```diff
@@ -156,3 +163,24 @@ AutoTSTrainer and TSPipeline accepts data frames as input. An exmaple data frame
  # ... do sth. e.g. incremental fitting
  loaded_ppl.save(another_file)
  ```
+### built-in models vs. AutoTS
+
+Now we show some comparison results between manually tuned models vs. AutoTS, using network traffic forecasting as an example.  
+
+We first trained an LSTMForecaster with 5 epochs (*Experiment 1*). Then we use AutoTS to train the model and find out the best configuration out of 120 trials (*Experiment 2*). The final best pipeline was acutally traied 50 epochs before it stops. To make the comparison fairer, we also train 50 epochs for LSTMForecaster with the same hyper parameters (*Experiment 3*). The time and accuracy results are as shown in below table. AutoTS achieves the best accuracy, while the time is still acceptable (on single node with 2 trials runing in parallel at the same time)
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/README.md:170]: 1) why do we need Experiment 1? We can remove it.
2) Remove time and single node

File: pyzoo/zoo/zouwu/README.md [hunk 3]
```diff
@@ -156,3 +163,24 @@ AutoTSTrainer and TSPipeline accepts data frames as input. An exmaple data frame
  # ... do sth. e.g. incremental fitting
  loaded_ppl.save(another_file)
  ```
+### built-in models vs. AutoTS
+
+Now we show some comparison results between manually tuned models vs. AutoTS, using network traffic forecasting as an example.  
+
+We first trained an LSTMForecaster with 5 epochs (*Experiment 1*). Then we use AutoTS to train the model and find out the best configuration out of 120 trials (*Experiment 2*). The final best pipeline was acutally traied 50 epochs before it stops. To make the comparison fairer, we also train 50 epochs for LSTMForecaster with the same hyper parameters (*Experiment 3*). The time and accuracy results are as shown in below table. AutoTS achieves the best accuracy, while the time is still acceptable (on single node with 2 trials runing in parallel at the same time)  
+
+|Experiment No.|Model|Mean Squared Error (smaller the better)|Symmetric Mean Absolute Percentage Error (smaller the better)|Epochs|Training Time|
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/README.md:172]: remove time

**Comment by shane-huang** [comment]
updated, please kindly review again @jason-dai

File: pyzoo/zoo/zouwu/README.md [hunk 4]
```diff
@@ -165,22 +167,27 @@ AutoTSTrainer and TSPipeline accepts data frames as input. An exmaple data frame
...
+Here we show some comparison results between manually tuned built-in models vs. AutoTS, using [network traffic forecast case](https://github.com/intel-analytics/analytics-zoo/tree/master/pyzoo/zoo/zouwu/use-case/network_traffic) as an example.  In paritcular, we compare    
+*  manually tuned built-in LSTMForecaster
+*  auto-tuned LSTM-based pipeline using AutoTS (obtained out of ~100 trials)
 
-We first trained an LSTMForecaster with 5 epochs (*Experiment 1*). Then we use AutoTS to train the model and find out the best configuration out of 120 trials (*Experiment 2*). The final best pipeline was acutally traied 50 epochs before it stops. To make the comparison fairer, we also train 50 epochs for LSTMForecaster with the same hyper parameters (*Experiment 3*). The time and accuracy results are as shown in below table. AutoTS achieves the best accuracy, while the time is still acceptable (on single node with 2 trials runing in parallel at the same time)  
+From below results, we can see that the features selected by AutoTS make much sense in our case, and AutoTS does achieved much better accuracy results than manually tuned model.
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/README.md:174]: "does achieve"

File: pyzoo/zoo/zouwu/README.md [hunk 5]
```diff
@@ -165,22 +167,27 @@ AutoTSTrainer and TSPipeline accepts data frames as input. An exmaple data frame
...
 
-|Experiment No.|Model|Mean Squared Error (smaller the better)|Symmetric Mean Absolute Percentage Error (smaller the better)|Epochs|Training Time|
-|-|--|-----|----|---|----|
-|1|Manually Tuned (LSTMForecaster)|27277.36|18.22%|5|8.1s|
-|2|AutoTS (LSTMForecaster)|2792.22|5.80%|50|40mins (120 trails on single node w/ 2 parallel workers)|
-|3|Manually Tuned (LSTMForecaster)|6312.44|8.61%|50|1min 9s|
+#### Accuracy: manually tuned vs. AutoTS
 
+|Model|Mean Squared Error (smaller the better)|Symmetric Mean Absolute Percentage Error (smaller the better)|Trained Epochs|
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/README.md:178]: smaller is better

File: pyzoo/zoo/zouwu/README.md [hunk 6]
```diff
@@ -165,22 +167,27 @@ AutoTSTrainer and TSPipeline accepts data frames as input. An exmaple data frame
  ```
 ### built-in models vs. AutoTS
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/README.md:168]: Built-in models

File: pyzoo/zoo/zouwu/README.md [hunk 7]
```diff
@@ -165,22 +167,27 @@ AutoTSTrainer and TSPipeline accepts data frames as input. An exmaple data frame
...
+#### Accuracy: manually tuned vs. AutoTS
 
+|Model|Mean Squared Error (smaller the better)|Symmetric Mean Absolute Percentage Error (smaller the better)|Trained Epochs|
+|--|-----|----|---|
+|Manually Tuned LSTMForecaster|6312.44|8.61%|50|
+|AutoTS (LSTM model)|2792.22|5.80%|50|
 
-Below is a comparison between manually selected parameters and auto-tuned parameters form AutoTS. We can see the features selected by AutoTS make much sense in our case.  
+
+#### Hyper parameters: manually selected vs. by AutoTS.
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/README.md:184]: manually tuned vs. AutoTS

**Comment by shane-huang** [comment]
Revised as you requested @jason-dai
